e2e/test/scenarios/question/column-compare.cy.spec.ts

```diff
@@ -10,134 +10,798 @@ import {
+
+      verifyAggregations([
+        {
+          name: "Count (previous period)",
+          expression: "Offset(Count, -1)",
+        },
+        {
+          name: "Count (vs previous period)",
+          expression: "Count - Offset(Count, -1)",
+        },
+        {
+          name: "Count (% vs previous period)",
+          expression: "Count / Offset(Count, -1) - 1",
+        },
+      ]);
+      verifyColumns([
+        "Count (previous period)",
+        "Count (vs previous period)",
+        "Count (% vs previous period)",
+      ]);
+    });
+
+    it("breakout on non-datetime column", () => {
+      createQuestion(
+        { query: QUERY_MULTIPLE_AGGREGATIONS_NON_DATETIME_BREAKOUT },
+        { visitQuestion: true, wrapId: true, idAlias: "questionId" },
+      );
+
+      verifySummarizeText({
+        itemName: "Compare to previous rows ...",
+        step2Title: "Compare “Count” to previous rows",
+        step1Title: "Compare one of these to the previous rows",
+        offsetHelp: "rows above based on “Category”",
+      });
+
+      tableHeaderClick("Category");
+      verifyNoColumnCompareShortcut();
+
+      verifyColumnDrillText({
+        itemName: "Compare “Count” to previous rows",
+        step2Title: "Compare “Count” to previous rows",
+        offsetHelp: "rows above based on “Category”",
+      });
+
+      verifyPlusButtonText({
+        itemName: "Compare to previous rows",
+        step2Title: "Compare “Count” to previous rows",
+        step1Title: "Compare one of these to the previous rows",
+        offsetHelp: "rows above based on “Category”",
+      });
+
+      verifyNotebookText({
+        itemName: "Compare to previous rows ...",
+        step2Title: "Compare “Count” to previous rows",
+        step1Title: "Compare one of these to the previous rows",
+        offsetHelp: "rows above based on “Category”",
+      });
+
+      toggleColumnPickerItems(["Value difference"]);
+      popover().button("Done").click();
+
+      cy.get("@questionId").then(questionId => {
+        expectGoodSnowplowEvent({
+          event: "column_compare_via_shortcut",
+          custom_expressions_used: CUSTOM_EXPRESSIONS_USED,
+          database_id: SAMPLE_DB_ID,
+          question_id: questionId,
+        });
+      });
+
+      verifyAggregations([
+        {
+          name: "Count (previous value)",
+          expression: "Offset(Count, -1)",
+        },
+        {
+          name: "Count (vs previous value)",
+          expression: "Count - Offset(Count, -1)",
+        },
+        {
+          name: "Count (% vs previous value)",
+          expression: "Count / Offset(Count, -1) - 1",
+        },
+      ]);
+      verifyColumns([
+        "Count (previous value)",
+        "Count (vs previous value)",
+        "Count (% vs previous value)",
+      ]);
     });
   });
 });
 
-function toggleColumnPicker() {
+function toggleColumnPickerItems(names: string[]) {
+  cy.findByTestId("column-picker").click({ force: true });
```

Comment:
nit. no need for `force`, same for the line after `for of`

```suggestion
  cy.findByTestId("column-picker").parent().click();
```